frontend/src/pages/questions/components/CalculatorTool.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    Box,
+    Tooltip
+} from "@chakra-ui/react";
+import { BsCalculatorFill, BsCalculator } from "react-icons/bs";
+import { useEffect } from "react";
+
+type CalculatorContentProps = {
+    width?: string;
+    height?: string;
+}
+
+export const CalculatorContent = ({ width = "600px", height = "400px" }: CalculatorContentProps) => {
+    const url = "https://www.desmos.com/api/v1.6/calculator.js?apiKey=dcb31709b452b1cf9dc26972add0fda6";
```

Comment:
Lets move api key to the .env files

Reply:
API url is moved to `.env.development.local` and the file is added to gitignore